src/test/java/com/flightstats/hub/webhook/TagWebhookTest.java

```diff
@@ -54,7 +54,7 @@ private ChannelConfig createChannelConfig(String name, String tag) {
                 .build();
     }
 
-    @BeforeSuite
+    @Before
```

Comment:
If I remember correctly, junit's `@Before` runs before each test. `@BeforeClass` should be the equivalent of `@BeforeSuite`.  Junit5 is much more intuitive, with `@BeforeEach` and `@BeforeAll`.

Reply:
you are right Laurie. There is only one test in this class. So it does not matter. Junit's Before class requires Beforeclass methods to be static.

Reply:
Ah, I didn't realize there was only one test... Honestly I don't think I would have added a separate setup method for a single test. I know you didn't write this; just thinking out loud.